### PR TITLE
Update Blazor CMS guide with multitenancy support and interactivity demo

### PR DESCRIPTION
**Pull request by psijkof**
Extended the Blazor App Sample on OrchardCore.Samples, via PR: https://github.com/OrchardCMS/OrchardCore.Samples/pull/45

And extended the documentation that is waiting in this PR, here: https://github.com/OrchardCMS/OrchardCore/pull/15213

### CODE REVIEW

File: src/docs/guides/create-blazor-cms/README.md [hunk 1]
```diff
@@ -572,15 +573,15 @@ Our blazor application's basic structure is finished. Now we are ready to create
         
         @code
         {
-        	protected override async Task OnInitializedAsync()
-        	{
+         protected override async Task OnInitializedAsync()
```
Comment (hishamco) [inline src/docs/guides/create-blazor-cms/README.md:576]: Fix the formatting so the tab should be 4 spaces

**Comment by Piedone** [comment]
Awesome! @ns8482e would you be interested to review?

File: src/docs/guides/create-blazor-cms/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Introduction
 
-In this article, you will learn how to use Orchard CMS as a decoupled CMS with .NET Blazor Application.
+In this article, you will learn how to use Orchard CMS as a decoupled CMS with .NET Blazor (SSR/Server Interactive) Application.
```
Comment (hishamco) [inline src/docs/guides/create-blazor-cms/README.md:3]:
```suggestion
In this article, you will learn how to use Orchard CMS as a decoupled CMS with the .NET Blazor (SSR/Server Interactive) Application.
```

File: src/docs/guides/create-blazor-cms/README.md [hunk 3]
```diff
@@ -146,9 +146,10 @@ dotnet sln add ./BlazorCms
 
 In this section, we will create a .NET Blazor application, as Razor Class library and then, we will reference it in the Orchard Core application.
 
-!!! note
+> !!! NOTE
+> 
+> You can add `.razor` files in the main Orchard Core Web application, however adding `.razor` files in Orchard Core modules are not supported. For this reason, and to have maximum reusability of your razor components, always use the razor class library and add it as a reference to your Orchard Core Web project or Orchard Core module.
```
Comment (hishamco) [inline src/docs/guides/create-blazor-cms/README.md:151]:
```suggestion
> You can add `.razor` files in the main Orchard Core Web application, however adding `.razor` files in Orchard Core modules is not supported. For this reason, and to have maximum reusability of your razor components, always use the razor class library and add it as a reference to your Orchard Core Web project or Orchard Core module.
```

**Review by hishamco — CHANGES_REQUESTED** [review]

**Review by ns8482e — APPROVED** [review]

**Comment by agriffard** [comment]
Add the path to the .md file in mkdocs.yml, below the 'How to guides' node to make it appear in the menu and in order to build the page.

**Pull request merged**